### ISSUE TITLE
you really like the word whiteList

### ISSUE THREAD

**Issue by tippi-fifestarr** [cromewar/1w33k-smartContracts#1]
why not use the word allowList

**Comment by cromewar** [comment]
Do not seem relevant for me.

**Comment by tippi-fifestarr** [comment]
The color of the list does not have relevance to its function (allowing or approving or saying yes to certain addresses), this naming choice based on your preference towards the traditional "whiteList" creates technical debt in non-coders and non-native English speakers.  being cryptic or mysterious should be a choice, not an accident.

**Comment by n4n0b1t3** [comment]
yeah, Tippi is right. I am just not sensitive to the topic of racially loaded language and I definitely should. @cromewar  [more info](https://abusix.com/resources/blocklists/why-changing-the-terms-blacklist-and-whitelist-isnt-as-easy-as-it-might-seem/#:~:text=Still%2C%20it%20just%20gets%20way,of%20approved%20or%20favored%20items.)
